python/server/qgsmsutils.sip

```diff
@@ -0,0 +1,34 @@
+/***************************************************************************
+    qgsmsutils.h
+    ---------------------
+    begin                : August 2010
+    copyright            : (C) 2010 by Marco Hugentobler
+    email                : marco dot hugentobler at sourcepole dot ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+%ModuleHeaderCode
+#include <QString>
+%End
+
+//! Some utility functions that may be included from everywhere in the code
+namespace QgsMSUtils
+{
+%TypeHeaderCode
+#include <qgsmsutils.h>
+%End
+
+  struct QgsLayerCoordinateTransform
+  {
+    QString srcAuthId;
```

Comment:
As above, should be `sourceAuthId` and `destinationAuthId`. (As explanation we are doing this to make the api more predictable  - in the past we mix up things like dst/dest/destination which means you're never sure which to use without having to check the docs.